guides/welfare/state_of_mind.md

```diff
@@ -0,0 +1,15 @@
+# State of Mind
+
+We want every one to be happy, including our team. To that end we try and provide [flexible ways of working](https://github.com/madetech/handbook/tree/master/benefits) so that we can have a healthy work/life balance.
+
+Sometimes though, personal circumstances, stress at work, and other factors can affect our state of mind. Rather than consider this shameful, or brushing it under the carpet, we want to recognise that we aren't always going to be happy and nor should we be expect to be.
+
+## Supporting each other
+
+We are a team and it is our duty to care for each other and ourselves, the latter being just as important as the former.
+
+- Take time off when you're not feeling great in the mind
+- Ask a colleague, friend, family member for support or lend an ear if you can, sharing helps
+- If a colleague looks like they're struggling or need help, be extra kind and supportive, ask if they need help
+- If you want confidential advice or phone counselling you can use our [employee assistance programme](employee_assistance.md)
+- If you feel like you're really in trouble, please ask a director or colleague for help, we can provide assistance in whatever shape it need take, from extending time off to providing a course of counselling
```

Comment:
I feel like this point, the second and the third are the same (ask for/ offer help).